backend/api/views.py

```
@@ -1,96 +1,75 @@
+        })
 
-class SurveyResponseViewSet(viewsets.ModelViewSet):
-    queryset = SurveyResponse.objects.all()
-    serializer_class = SurveyResponseSerializer
+class OnboardingAPIView(APIView):
+    permission_classes = [IsAuthenticated]
     
-    # Removed permission_classes
-    def get_queryset(self):
-        return SurveyResponse.objects.filter(user=self.request.user)
+    def post(self, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `post`

(ANN201)